release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -149,6 +149,16 @@ You can now use CSI to specify volumes directly in the Pod specification, rather
 
 Volume cloning using CSI, previously in Technology Preview, is now fully supported in OpenShift Container Platform 4.5. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
 
+[id="ocp-4-5-operators"]
+=== Operators
+
+[id="ocp-4-5-metering-operator"]
+==== Metering Operator
+
+You can now upgrade a Metering Operator to 4.5 from 4.2 through 4.4. Previously, you had to uninstall your current metering installation and then reinstall the new version of the Metering Operator. //For more information, see ../metering/metering-upgrading-metering.adoc#upgrading-metering[Upgrading Metering].
+
+With this update, support for respecting a cluster-wide proxy configuration was also added. Additionally, the upstream repository was moved from the operator-framework organization to kube-reporting.
```

Comment:
I usually try to avoid past tense when describing new updates, if it makes sense. Maybe:

s/was also added/is now available
s/was moved/moved